src/main/java/com/sourcegraph/cody/agent/CodyAgentClient.java

```diff
@@ -97,13 +97,13 @@ public CompletableFuture<Boolean> textDocumentShow(TextDocumentShowParams params
     return acceptOnEventThread("textDocument/show", onTextDocumentShow, params);
   }
 
-  @JsonRequest("textDocument/openUntitledDocument")
-  public CompletableFuture<Boolean> openUntitledDocument(UntitledTextDocument params) {
-    if (onOpenUntitledDocument == null) {
-      return CompletableFuture.failedFuture(
-          new Exception("No callback registered for textDocument/openUntitledDocument"));
+  @JsonRequest("textDocument/openDocument")
```

Comment:
Note: I think we need clearer names to distinguish

- textDocument/open
- textDocument/openDocument

We can do this separately after GA.